core/src/main/java/com/netflix/conductor/service/TaskService.java

```
@@ -152,7 +152,14 @@ public String updateTask(TaskResult taskResult) {
     public String ackTaskReceived(String taskId, String workerId) {
         ServiceUtils.checkNotNullOrEmpty(taskId, "TaskId cannot be null or empty.");
         LOGGER.debug("Ack received for task: {} from worker: {}", taskId, workerId);
-        return String.valueOf(executionService.ackTaskReceived(taskId));
+        boolean ackResult;
+        try {
+            ackResult = executionService.ackTaskReceived(taskId);
+        } catch (Exception e) {
```

Write a comment explaining rational behind swallowing exception here.

---

done